hardware/display/README.md

```diff
@@ -14,6 +14,26 @@ The DSI display is designed to work with Raspberry Pi models 2, 3 and 4 that hav
 
 LCD displays have an optimum viewing angle, and depending on how the screen is mounted it may be necessary to change the orientation of the display to give the best results. By default, the Raspberry Pi Touch Display and Raspberry Pi are set up to work best when viewed from slightly above, for example on a desktop. If viewing from below, you can physically rotate the display, and then tell the system software to compensate by running the screen upside down.
 
+### FKMS Mode
+
+FKMS mode is used by default on the Raspberry Pi 4B. FKMS uses the DRM/MESA libraries to provide graphics and 3D acceleration. 
+
+To set screen orientation when running the graphics desktop, select the `Screen Configuration` option from the `Preferences` menu. Right Click on the DSI display rectangle in the layout editor, select Orientation then the required option.
+
+To set screen orientation when in console mode, you will need to edit the kernel command line to pass the required orientation to the system. 
+```bash
+sudo nano /boot/cmdline
```

Comment:
Should this be `/boot/cmdline.txt` ?

Reply:
Did I mention about the need for copy editors!!!!